v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -185,14 +235,48 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
   void setIgnoreColumns(String value);
 
   @TemplateParameter.Text(
-      order = 15,
+      order = 18,
       optional = true,
       description = "Maximum number of connections to Source database per worker",
       helpText =
-          "Configures the JDBC connection pool on each worker with maximum number of connections. Use a negative number for no limit. Default value is 100.",
+          "Configures the JDBC connection pool on each worker with maximum number of connections. Use a negative number for no limit.",
```

Comment:
unlimited is a bad idea as it may overwhelm the source db, especially in case of bugs. Let's set a reasonable max value here

Reply:
Our default is 100.
Unlimited is an option jdbc exposes with -ve value here. Just documented the behaviour.